src/lib/Components/Contact.svelte

```diff
@@ -0,0 +1,83 @@
+<script>
+	import {
+		IconMailFilled,
+		IconPhoneFilled,
+		IconMapPinFilled,
+	} from '@tabler/icons-svelte';
+
+	let firstname = '';
+	let lastname = '';
+	let email = '';
+	let phone = '';
+	let message = '';
+
+	const handleSubmit = () => {
+		console.log('First Name:', firstname);
+		console.log('Last Name:', lastname);
+		console.log('Email:', email);
+		console.log('Phone:', phone);
+		console.log('Message:', message);
+	};
+</script>
+
+<section class="py-10 bg-surface-300">
```

Comment:
Are you able to make the color scheme of your section more consistent with the theme its in please? See Hero Section for example.

Reply:
E.g. Section background color remains the same but should be darker (including header text as a lighter text color).
The form part remains white with black text color, but should be darker background color for form part of contact section, and lighter text color.